lib/airport.rb

```diff
@@ -2,22 +2,29 @@
 
 class Airport
 
-  # def iinitialize
-  #   @plane = []
-  # end
+  DEFAULT_CAPACITY = 20
+  def initialize
+    @planes = Array.new
+  end
 
   def plane_landing
-    Plane.new
+    if !full 
+      plane = Plane.new
```

Comment:
Would it make sense to pass in the plane, as an arg, rather than create one inside this method?